backend/api/controllers/OrderController.js

```diff
@@ -115,48 +144,50 @@ const findProducts = async ({ ...params }) => {
  */
 const processOrder = ({ ...params }) => {
   const calculatePrice = {
-    NO_SALE: (item) => {
-      return item.quantity * item.price;
+    NO_SALE: ({ quantity, product }) => {
+      return quantity * product.price;
     },
 
-    PRICE_MOD: (item) => {
-      return item.quantity * (item.price * item.price_mod).toFixed(2);
+    PRICE_MOD: ({ quantity, product }) => {
+      return quantity * (product.price * product.price_mod).toFixed(2);
     },
 
-    PACKAGE: (item) => {
-      const discount = Math.floor(item.quantity / item.package_get_count) * item.package_pay_count * item.price;
-      const remainder = (item.quantity % item.package_get_count) * item.price;
-      return discount + remainder;
+    PACKAGE: ({ quantity, product }) => {
+      const discounted = Math.floor(quantity / product.package_get_count) * product.package_pay_count * product.price;
```

Comment:
Can destructure more to reduce length of line.
const { package_get_count, package_pay_count, price } = product;